File: algorithms/03_search/linear_search/linear_search.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {

  // input
  int N, v;
  cin >> N >> v;
  vector<int> a(N);
  for (int i = 0; i < N; ++i)
    cin >> a[i];

  // linear search
  bool exist = false;
  for (int i; i < N; ++i) {
    if (a[i] == v) {
      exist = true;
    }
  }

  // print
  if (exist)
    cout << "Yes" << endl;
  else
    cout << "No" << endl;
}

File: algorithms/03_search/linear_search/linear_search.kt

import java.util.*

fun main() {
    val scanner = Scanner(System.`in`)

    // input
    val n = scanner.nextInt()
    val v = scanner.nextInt()
    val a = IntArray(n) { scanner.nextInt() }

    // linear search
    var exist = false
    for (element in a) {
        if (element == v) {
            exist = true
            break
        }
    }

    // print
    if (exist) {
        println("Yes")
    } else {
        println("No")
    }
}
